docs/components/board/_index.md

```diff
@@ -0,0 +1,1565 @@
+The GPIO pins on SBCs are generally not accessible to external computers.
+{{% /alert %}}
+
+**GPIO Peripheral and Computer**:
+
+A device with GPIO pins, like the [numato](numato) model of board, can still act as the signal wire hub of your robot.
+However, a board like this does not contain a computer to run `viam-server` on the robot, so it can only act as the software hub if you have physically connected it to another computer.
+
+In this case, the computer running `viam-server` signals through the GPIO peripheral's GPIO pins to communicate with the other hardware components of the robot.
+
+{{% figure src="img/board-comp-options.png" alt="Image showing two board options: First, running viam-server locally and second, running via a peripheral plugged into the USB port of a computer that is running the viam-server." title="Two different board options: a single-board computer with GPIO pins running `viam-server` locally, or a GPIO peripheral plugged into a desktop computer's USB port, with the computer running `viam-server`." %}}
+
+## Configuration
+
+For configuration information, first click on one of the following models:
+
+| Model | Description |
+| ----- | ----------- |
+| [`pi`](pi) | [Raspberry Pi 4](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/) or [Raspberry Pi Zero 2 W](https://www.raspberrypi.com/products/raspberry-pi-zero-2-w/) |
+| [`ti`](ti) | [Texas Instruments TDA4VM](https://devices.amazonaws.com/detail/a3G8a00000E2QErEAN/TI-TDA4VM-Starter-Kit-for-Edge-AI-vision-systems) |
+| [`beaglebone`](beaglebone) | [BeagleBoard's BeagleBone AI 64](https://beagleboard.org/ai-64) |
+| [`jetson`](jetson) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/), [NVIDIA Jetson  Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
+| [`nanopi`](nanopi) | [FriendlyElec’s NanoPi Mini Board](https://www.friendlyelec.com/index.php?route=product/category&path=69) |
+| [`numato`](numato) | [Numato GPIO Modules](https://numato.com/product-category/automation/gpio-modules/), peripherals for adding GPIO pins |
+| [`pca9685`](pca9685) | [PCA9685 Arduino I<sup>2</sup>C Interface](https://www.adafruit.com/product/815), a 16-channel [I<sup>2</sup>C](#i2cs) [servo](/components/servo) driver peripheral |
+| [`fake`](fake) | A model used for testing, with no physical hardware |
+
+The following configuration attributes are available for every *board* model besides the `numato` and `pca9685` peripherals and `fake`.
+
+Configuring these attributes on your board allows you to integrate [analog-to-digital converters](#analogs), [digital interrupts](#digital_interrupts), and components that must communicate through the [SPI](#spis) and [I<sup>2</sup>C](#i2cs) protocols into your robot.
+
+### `analogs`
```

Comment:
I’d propose making all these just sentence case, not matching the attribute names, because they’re general concepts not just attributes, it’d look nicer, and the attribute names are clear in the information under the headings anyway.

Reply:
I see your point but I think this makes it a lot easier to parse with the way people are directed between the models and the main page. It makes leafing through configuration attributes more simple. I also like the way it looks personally haha

Reply:
I think it is harder to tell that they're headings, and they appear blue in the right nav :o

Reply:
how they appear in the side nave can be fixed regardless of sentence case. I think we have it in a lot of cases like this because sentence case doesn't always work well